golos.publication/golos.publication.cpp

```diff
@@ -849,11 +862,13 @@ void publication::reblog(name rebloger, structures::mssgid message_id) {
 
 int64_t publication::pay_delegators(int64_t claim, name voter, 
         eosio::symbol tokensymbol, std::vector<structures::delegate_voter> delegate_list) {
+    posting_params_singleton cfg(_self, _self.value);
+    const auto &vesting_acc_param = cfg.get().vesting_acc_param;
```

Comment:
actually, we don't need `vesting_acc_param`, but need `vesting_acc_param.account` value. propose to store it in variable instead of wrapper structure